transcom/mymove
# MB-16395 - Create sit bundles

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16395)

## Summary

This PR should create two functions that create bundles of SIT service items.
Details to give additional scrutiny:
- The bundle builder functions don't follow the same customs/traits pattern as many other builder functions since they are building more than one model; the intention is for them to be simple and convenient at the cost of some customizability
- The `BuildRealMTOServiceItemWithAllDeps` function has been updated to pass customs/traits
- The default sit allowance in the test scenarios has been changed from 200 to 90 to better reflect real scenarios
## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the office app as a TOO
2. Create a move with a SIT test harness scenario
3. Navigate to the move's MTO tab and verify that the SIT display looks generally correct

SIT TestHarness Scenarios to test:
- HHGMoveInSIT
- HHGMoveInSITDeparted
- HHGMoveInSITEndsToday
- HHGMoveInSITEndsTomorrow
- HHGMoveInSITEndsYesterday
- HHGMoveInSITNotApproved
- HHGMoveInSITStartsInFuture
- HHGMoveInSITWithAddressChangeRequestOver50Miles
- HHGMoveInSITWithAddressChangeRequestUnder50Miles
- HHGMoveInSITWithPendingExtension

### Frontend


### Backend
